dataactcore/scripts/pullFPDSData.py

```diff
@@ -59,6 +64,33 @@ def extract_text(data_val):
     return data_val
 
 
+def is_valid_zip(zip_code):
+    if re.match('^\d{5}(-?\d{4})?$', zip_code):
+        return True
+    return False
+
+
+def get_county_by_zip(sess, zip_code):
```

Comment:
`is_valid_zip` accounts for hyphenated zips (00000-0000), but it would never get past any of the `if` statements including `len(zip_code)` because they only allow a length of 9 or 5

Reply:
See? This is why I need extra sets of eyes. I changed how I handled the checking if it was valid part and didn't change the length checks.